tests/jballerina-unit-test/src/test/resources/test-src/expressions/builtinoperations/clone-operation.bal

```diff
@@ -384,25 +385,22 @@ public function testCloneArrayWithError() returns boolean {
 
 public function testCloneMapWithError() returns boolean {
     map<error> errMap = {
-        e1: err1,
-        e2: err2,
-        e3: err3,
-        e4: err4
```

Comment:
Any particular reason to change this?

Reply:
I really don't remember why I did this, removed the change